csv_parser/csv_parser.py

```diff
@@ -680,6 +690,19 @@ def DFS(root, use_elem):
 
     print(f'{Color.BOLD}{Color.UNDERLINE}{Color.PURPLE}Generating JSON schema...{Color.END}')
     DFS(rootObject, build_json_schema)
+
+    """Before dumping, we verify the json schema definitions to make sure no objects with no properties are defined.
+    This verification is skipped for RS-ERROR schema"""
+    if MODEL_TYPE != "error":
+        empty_object_errors = []
+        for key in json_schema['definitions']:
+            if not json_schema['definitions'][key]['properties']:
```

Comment:
```suggestion
        for key, definition in json_schema['definitions'].items():
            if not definition['properties']:
```
Peut-être un poil plus élégant ?

Reply:
Et pas besoin de checker qu'il existe mais est vide ? Tu as fait un test avec TECHNICAL ou autre, ça plante bien comme voulu ?

Reply:
T'as raison, c'est pas nécessaire; j'avais un erreur de structure dans le schéma technical (je redefenissais la propriété level3Data, exactement ce qu'on veut pas faire) et ça m'a fait croire que le problème est lié a la logique ici

Reply:
Selon ce que j'ai vu en testant la propriété 'properties' n'existe pas du tout quand l'objet n'a pas d'enfants.
Ca plante comme anticipé: 
![image](https://github.com/user-attachments/assets/567aa679-36ad-4472-8eb3-b138730c1e8d)